src/react/PropTypes.js

```diff
@@ -1,14 +1,12 @@
-import {PropTypes} from "react";
+import PropTypes from "prop-types";
 
```

Comment:
Any reason why we can't keep
```
const {shape, object, string} = PropTypes;
```
the same? Or is this the new way Facebook suggests to use the PropTypes package?